front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState } from 'react';
+
+import { Checkbox } from '@osrd-project/ui-core';
+import { ChevronDown, ChevronRight, Clock, Flame, Manchette } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+
+import type { PacedTrainId } from 'reducers/osrdconf/types';
+import { ms2min } from 'utils/timeManipulation';
+
+import TimetableItemActions from '../TimetableItemActions';
+import type { PacedTrainWithResult } from '../types';
+
+type PacedTrainItemProps = {
+  isInSelection: boolean;
+  handleSelectPacedTrain: (pacedTrainId: PacedTrainId) => void;
+  pacedTrain: PacedTrainWithResult;
+  isOnEdit: boolean;
+  isProjectionPathUsed: boolean;
+  selectPacedTrainToEdit: (pacedTrain: PacedTrainWithResult) => void;
+};
+
+const PacedTrainItem = ({
+  isInSelection,
+  handleSelectPacedTrain,
+  pacedTrain,
+  isOnEdit,
+  isProjectionPathUsed,
+  selectPacedTrainToEdit,
+}: PacedTrainItemProps) => {
+  const { t } = useTranslation(['operationalStudies/scenario']);
+
+  const [isOccurrencesListOpen, setIsOccurrencesListOpen] = useState(false);
+
+  const toggleOccurrencesList = () => setIsOccurrencesListOpen((open) => !open);
+  const selectPathProjection = async () => {};
+  const duplicatePacedTrain = async () => {};
+  const deletePacedTrain = async () => {};
+
+  const pacedTrainCadence = pacedTrain.paced.step;
+
+  const occurrencesCount = Math.floor(pacedTrain.paced.duration.ms / pacedTrain.paced.step.ms);
```

Comment:
If we have a duration at PT120M and step at PT29M, we should have 5 occurrences not 4.
```suggestion
  const occurrencesCount = Math.ceil(pacedTrain.paced.duration.ms / pacedTrain.paced.step.ms);
```

Reply:
Nice, thank you. Done